nifty-core/src/main/java/com/facebook/nifty/ssl/SslSession.java

```diff
@@ -65,4 +65,13 @@ public long getEstablishedTime() {
     public X509Certificate getPeerCert() {
         return peerCert;
     }
+
+    public String toString() {
+        return "SslSession(alpn=" + (alpn == null ? "null" : alpn) +
+            ", npn=" + (npn == null ? "null" : npn) +
+            ", version=" + (version == null ? "null" : version) +
+            ", cipher=" + (cipher == null ? "null" : cipher) +
+            ", establishedTime=" + Long.toString(establishedTime) +
+            ", peerCert=" + (peerCert == null ? "null" : peerCert.toString()) + ")";
```

Comment:
No need to check for null and explicitly calling toString. 
I think the following should do the right thing.
", peerCert=" + peerCert + ")";